ocamldoc/odoc_html.ml

```diff
@@ -2396,7 +2397,7 @@ class html =
         bs b "<html>\n";
         self#print_header b (self#inner_title title);
         bs b "<body>\n";
-        self#print_navbar b None None "";
+        if !show_navbar then self#print_navbar b None None "";
```

Comment:
Perhaps put the `if` in `print_navbar` itself (and, if the idea of a function called `print_` which may not print offends, rename the method to `process_navbar`/`do_navbar` or something).